apps/frontend/src/lib/store/block.store.ts

```
@@ -21,10 +21,13 @@
   isLoading: boolean
   error: string | null
   fetchBlocks: (
-    session: string,
+    session: string | Promise<string>,
     spaceId: string
   ) => Promise<FetchBlocksResult | void>
```

Biome (1.9.4)

[error] void is confusing inside a union type.

Unsafe fix: Use undefined instead.


(lint/suspicious/noConfusingVoidType)